src/displayapp/screens/FlashLight.cpp

```diff
@@ -56,23 +57,20 @@ FlashLight::~FlashLight() {
   systemTask.PushMessage(Pinetime::System::Messages::EnableSleeping);
 }
 
+void FlashLight::SetBrightness() {
+  brightnessController.Set(isOn ? brightnessLevel : std::min(brightnessLevel, maxBrightnessLevel));
```

Comment:
suggestion: instead of limiting the off-brightness, why even change it? Let's keep it constant and only change the brightness when the flashlight is on?

Reply:
This is to minimize the light leakage when the flashlight is off and using a low brightness setting. I wonder if it could be always low in the off position. Is this what you mean?

Reply:
not quite, I propose to use the brightnesslevel from the system when the flashlight is turned off (instead of using the setting as upper bound). And only use the flashlight-set-brightnesslevel when the flashlight is on. This would reduce code complexity even more.

With the current upper bound approach the brightness of the off-flashlight may change, but not always. I think this could be confusing (even writing about it to explain the behaviour is rather hard and need much explanation)

Reply:
I don't think it's a good idea to just use the system backlight level because of the light leakage. Using medium when off and low when on for example doesn't make much sense. Keeping it always low would lower the visibility in daylight, but I guess you wouldn't use a flashlight then anyway.

Reply:
I don't know why it would be confusing. The brightness is optimal in each situation, it only changes inside the flashlight app and returns to normal when exiting, however now I'm leaning towards making it always low. Shall I make the change here?

Reply:
Always low when off is the most reasonable, given that it is used in dark surroundings. It would then be weird, when one accidentally opened the flashlight in bright daylight, but this should be ok. 
I like the always low approach. 👍

Reply:
Ilike the always low when off solution as well